action.yaml

```diff
@@ -108,7 +106,7 @@ runs:
           else
               html_url=""
           fi
-          
+          echo "$html_url"
```

Comment:
Is this echo needed?

Reply:
Before this step was added by @AdityaHitachi but in my view it's not required .If want I will remove

Reply:
@bsairamhitachi I added that for debugging purposes. You can remove it. There might be other "echo" commands for debugging that I have added. Review the code and remove all of them.